shared/overview/2.x.md

```diff
@@ -12,40 +12,41 @@ The first version of {{ $names.os.lower }} was developed as part of the {{ $name
 
 We look forward to working with the community to grow and mature {{ $names.os.lower }} into an operating system with even broader device support, a broader operating envelope, and as always, taking advantage of the most modern developments in security and reliability.
 
-## Variants of {{ $names.os.lower }}
+### Development vs. Production mode
 
-Each version of {{ $names.os.lower }} is available in development and production variants, both built from the same source, but with slightly differing feature sets.
+{{ $names.os.lower }} can be downloaded in production or development mode. This can be later changed via [developmentMode][config-json-developmentmode]
```

Comment:
```suggestion
{{ $names.os.lower }} can be downloaded in production or development mode. This can be later changed via [developmentMode][config-json-developmentmode].
```